pages/common/perl.md

```diff
@@ -21,3 +21,11 @@
 - Run a Perl script in debug mode, using `perldebug`:
 
 `perl -d {{script.pl}}`
+
+- Replace all occurrences of a string in a file with another string, overwriting the file in-place:
+
+`perl -p -i -e 's/{{find}}/{{replace}}/g' {{filename}}`
+
+- Same, saving an unmodified copy of the original file with a different extension:
```

Comment:
"Replace all occurrences of a string in a file with another string, saving the original file with a given extension" - is this too long ?